Maui.RevenueCat.InAppBilling/Platforms/Android/Extensions/OwnershipTypeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Maui.RevenueCat.InAppBilling.Enums;
+using Com.Revenuecat.Purchases;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.Android.Extensions;
+internal static class OwnershipTypeExtensions
+{
+    internal static OwnershipId ToOwnershipId(this OwnershipType owner)
+    {
+        if (owner == OwnershipType.FamilyShared) return OwnershipId.FamilyShared;
+        if (owner == OwnershipType.Purchased) return OwnershipId.Purchased;
+        return OwnershipId.Unknown;
```

Comment:
- I would prefer `switch` here
- please rename `owner` to reflect the type used -> `ownershipType`

Reply:
I would also prefer switch, but you can't use a switch when the case values are not constants.  For example, OwnershipType.FamilyShared is not a constant value.  

Naming changed in ios and android